### PR TITLE
Editorial chapter 7

### PR DESCRIPTION
**Pull request by mlagally**

### CODE REVIEW

**Comment by mkovatsc** [comment]
This is an extremely messy PR looking at the commits. What is actually changing?

File: index.html [hunk 1]
```diff
@@ -1524,24 +1525,28 @@ <h2>Overview</h2>
                 <span class="rfc2119-assertion" id="arch-td-consumers-process">
                   <a>Consumers</a> MUST be able to process the <a>WoT Thing Description</a> format, which is based on JSON [[!RFC8259]].
                 </span>
-                The format can be processed either through classic JSON libraries or a JSON-LD processor, as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
+                The format can be processed either through classic JSON libraries or a JSON-LD processor, 
+                as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
                 A <a>TD</a> is instance-specific (i.e., describes an individual Thing, not types of Things)
-                and is the default Web representation of a <a>Thing</a>.
+                and is the default external textual (Web-) representation of a <a>Thing</a>.
```
Comment (mkovatsc) [inline index.html]: What does external mean?
What is an internal textual representation?
Why a hyphen after Web?

**Comment by mkovatsc** [comment]
Cannot review something that overlaps with many other reviews...

File: index.html (hunk 1 above)
Comment (mlagally) [reply index.html]: An internal representation can be anything - external means that it is available outside of the thing.

Comment (mkovatsc) [reply index.html]: then
external, textual
(needs comma)

**Comment by mlagally** [comment]
approved to merge in call on 7.5.

**Pull request closed without merging**